desktop/apps/editorial_features/components/venice_2017/client/index.coffee

```diff
@@ -45,6 +47,12 @@ module.exports = class VeniceView extends Backbone.View
     @VeniceVideoView.trigger 'swapVideo',
```

Comment:
This is currently failing for me -- 
![screen shot 2017-04-21 at 8 59 03 am](https://cloud.githubusercontent.com/assets/1497424/25278535/6339324c-2671-11e7-9bf1-f14b6b5a3fb8.png)